front/src/locales/en.json

```diff
@@ -400,6 +400,26 @@
       },
       "settings": {
         "title": "Settings"
+      },
+      "files": {
+        "title": "Files and multimedia contents",
+        "columns": {
+          "filename": "File",
+          "size": "Size",
+          "createdAt": "Created at",
+          "updatedAt": "Updated at"
+        },
+        "removeConfirmationDialog": {
+          "message": "Do you really want to remove file {file} ? It will also be unlinked from records of your database.",
```

Comment:
Proposition:
```suggestion
          "message": "Do you really want to remove file {file} ? It will also be remove from records of your database.",
```